# Resumo da secção - Key takeaways

# Exercício 3

# Qual é o resultado esperado do seguinte código?

s1 = '12.8'
i = int(s1)
s2 = str(i)
f = float(s2)
print(s1 == s2)

# Resposta: False
# Resposta correta: O código dispara uma exceção 'ValueError'
